public/video-ui/src/reducers/composerPageReducer.js

```diff
@@ -1,10 +1,10 @@
 export default function usage(state = {}, action) {
   switch (action.type) {
-    case 'VIDEO_PAGE_CREATE_POST_RECEIVE': {
-      const videoId = action.newPage.videoId;
+    case 'usage/addNewlyCreatedVideoUsage': {
```

Comment:
Small improvement, you can reference the action type with `addNewlyCreatedVideoUsage.type`